### PR TITLE
Fix table entries in documentation

### PR DESCRIPTION
**Pull request by mercur3**
In the current state the description for `/api/user/me` endpoint corresponds to `/api/admin` and vice versa.

### CODE REVIEW

File: docs/src/main/asciidoc/security-getting-started.adoc [hunk 1]
```diff
@@ -23,9 +23,9 @@ The steps in this tutorial guide you through building an application that provid
 |===
 |Endpoint | Description 
 |`/api/public`| The `/api/public` endpoint can be accessed anonymously.
-|`/api/users/me`|The `/api/admin` endpoint is protected with role-based access control (RBAC), and only users who have been granted the `admin` role can access it. 
+|`/api/users/me`| The `/api/users/me` endpoint is protected with RBAC and only users that have been granted the `user` role can access it. A JSON document with details about the user is returned as a response.
+| `/api/admin`| The `/api/admin` endpoint is protected with role-based access control (RBAC), and only users who have been granted the `admin` role can access it.
 At this endpoint, the `@RolesAllowed` annotation is used to declaratively enforce the access constraint.
```
Comment (gsmet) [inline docs/src/main/asciidoc/security-getting-started.adoc:28]: Maybe put this entry first given it arrives first in the doc?

**Review by gsmet — COMMENTED** [review]
Thanks for taking the time to create a pull request. I added a small suggestion.

**Review by gsmet — APPROVED** [review]
I adjusted it. Let's wait for CI and merge.

**Comment by mercur3** [comment]
Thanks, I was using the work device so I couldn't do it right away.

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**